src/components/EntityInfo.vue

```diff
@@ -180,6 +197,30 @@ export default class EntityInfo extends Mixins(EnumMixin) {
     // assume Business Profile URL is always reachable
     window.location.assign(businessProfileUrl)
   }
+
+  /** Get route breadcrumbs. */
+  private getBreadcrumbs (): Array<BreadcrumbInterface> {
+    const breadcrumbs = this.$route?.meta?.breadcrumb
+
+    // Apply the filing year to the breadcrumb trail for Annual Reports
+    let ArCrumb = breadcrumbs?.find(item => item.to.name === ANNUAL_REPORT)
```

Comment:
This could be a const since you never change its value.

Reply:
Nice solution BTW.

Reply:
Thanks, will update this.